addons/mail/static/src/web/activity/activity.js

```diff
@@ -51,6 +51,60 @@ export class Activity extends Component {
         this.attachmentUploader = useAttachmentUploader(this.thread);
     }
 
+    /**
+     * @returns {string}
+     */
+    get activityInfo() {
+        return markup(
+            sprintf(
+                _t(
+                    `<span class="fw-bolder %(classes)s }}">%(delay)s:</span> %(activity name)s <span class="o-mail-Activity-user">for %(responsible employee)s</span>`
```

Comment:
I would keep all classes out of translation if possible, it looks like a maintenance nightmare to have them inside.

Even html tags are annoying, but a bit less so.